README.md

```diff
@@ -74,6 +74,8 @@ Use these environment variables to change the default behaviour of the container
 |`-e DECONZ_VNC_MODE=1`|Set this option to enable VNC access to the container to view the deCONZ ZigBee mesh|
 |`-e DECONZ_VNC_PORT=5900`|Default port for VNC mode is 5900; this option can be used to change this port|
 |`-e DECONZ_VNC_PASSWORD=changeme`|Default password for VNC mode is 'changeme'; this option can (should) be used to change the default password|
+|`-e DECONZ_VNC_PASSWORD_FILE=/var/run/secret`|Per default this is disabled and DECONZ_VNC_PASSWORD is used. More details on secrets can be found in the [corresponding section from the official documentation](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets) |
+|`-e DECONZ_NOVNC_PORT=6080`|Default port for noVNC is 6080; this option can be used to change this port; setting the port to `0` will disable the noVNC functionality|
```

Comment:
We should also alert the user in the README that the minimum value for this is 6080 since the container checks for it and exits if it is less than 6080. I think I do the same thing for the VNC_PORT later in the environment variables section.

Reply:
As this check is also for the port 5900 of DECONZ_VNC_PORT, I'll add it to both variables.